client/components/bookingButton.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import styles from '../styles/bookingButton.css';
+
+const BookingButton = () => {
+  return (
+    <div>
+      <button className={styles.button} type="button">
```

Comment:
Maybe space before and after the curly brackets for readability.

Reply:
I believe this goes against the AirBnB style guide I think that's expected for object destructuring